### PR TITLE
bugs fixed, I think :)

### PR DESCRIPTION
**Pull request by mschwartz**
I'm going to merge this into master

### CODE REVIEW

**Comment by mschwartz** [comment]
@grgur please review

I thought I merged into develop already, but the PR is still here.

File: src/Tablet/Dashboard/TheaterTile.js [hunk 1]
```diff
@@ -80,13 +81,18 @@ const TheaterTile = ({ title }) => {
...
 
   const renderCurrentDevice = () => {
+    console.warn("rcd", currentDevice.current);
     if (currentDevice.current === "TiVo") {
       return <TiVo device={deviceMap.tivo} />;
     } else if (currentDevice.current === "Apple TV") {
       console.log("atv", appleTV);
       return <AppleTV device={appleTV.device} />;
+    } else if (currentDevice.current === "AppleTV") {
+      console.log("atv", appleTV);
```
Comment (grgur) [inline src/Tablet/Dashboard/TheaterTile.js]: can we go without console.logs?

Comment (mschwartz) [reply src/Tablet/Dashboard/TheaterTile.js]: Yes, I think they were removed in the PR I merged to master ...

File: src/Tablet/Sensors/SensorsTab.js [hunk 2]
```diff
@@ -23,6 +23,20 @@ const SensorsTab = () => {
   });
   const types = ["contact", "motion", "battery", "temperature", "illuminance", "humidity"];
 
+  const clearSensors = () => {
+    const s = sensors.current;
```
Comment (grgur) [inline src/Tablet/Sensors/SensorsTab.js]: is it possible to not use `s` keyword? Once we start overhauling styles, we'll use `s` for css modules. 
You don't have to fix this now, we can fix when we cross that bridge

Comment (mschwartz) [reply src/Tablet/Sensors/SensorsTab.js]: within a function, variable names are "who cares?" to me

I chose s because it's short and stands for "sensors"

**Pull request merged**